Jenkinsfile

```diff
@@ -11,7 +11,7 @@ pipeline {
         stage('Check formatting') {
             steps {
                 echo 'Checking formatting...'
-                sh 'bash -c "tools/check_format.sh 2>&1 >(tee tools/check_format.txt)"'
+                sh 'bash -c "tools/check_format.sh -j8 2>&1 >(tee tools/check_format.txt)"'
```

Comment:
To fix the check_format script I've currently just hardcoded it to 8 jobs. Should this just be dependent on `nproc` or what?

Reply:
I recommend just having `check_format.sh` call `format.py` with `-j` on its own, without specifying a number of jobs, and format.py should default to the number of CPUs for the machine